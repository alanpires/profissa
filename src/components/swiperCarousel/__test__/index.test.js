import React from 'react'
import renderer from 'react-test-renderer'
import Carousel from '../index'

describe('Snapshot', () => {
    it('Carousel renders correctly', () => {
        const tree = renderer.create(<Carousel />).toJSON()

        expect(tree).toMatchSnapshot()
    })
})
