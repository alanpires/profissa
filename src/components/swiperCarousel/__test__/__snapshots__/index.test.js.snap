// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Carousel renders correctly 1`] = `
<div
  className="swiper-container sc-bdnylx cTaNEn"
>
  <div
    className="swiper-button-prev"
  />
  <div
    className="swiper-button-next"
  />
  <div
    className="swiper-pagination"
  />
  <div
    className="swiper-wrapper"
  />
</div>
`;
